scenarios/centos-9/edpm_build_images_ci.yml

```diff
@@ -0,0 +1,2 @@
+---
+ansible_user_dir: "{{ lookup('env', 'HOME') }}"
```

Comment:
@raukadah this is related to your issues isn't it?

Reply:
Yes, and we are going with this pr https://github.com/openstack-k8s-operators/ci-framework/pull/286 we can do similar thing here also.

Reply:
#286 is close to be merged - imho we can remove that parameter now?